GuessTheFlag/Controller/YellowBackgroundView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  YellowBackgroundView.swift
+//  GuessTheFlag
+//
+//  Created by Michel Bernardo on 11/07/21.
+//
+
+import UIKit
+
+class YellowBackgroundView: UIView {
+    
+    lazy var scoreTopicLabel = buildlScoreTopicLabel(with: "Correct Answers: ")
+    lazy var scorePointLabel = buildScorePointLabel(with: "10")
+    lazy var scorePointsView = buildScorePointsView()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureConstraints()
+        configureBackgroundView()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func configureConstraints() {
+        addSubview(scoreTopicLabel)
+        addSubview(scorePointsView)
+        scorePointsView.addSubview(scorePointLabel)
+        
+        NSLayoutConstraint.activate([
+            scoreTopicLabel.topAnchor.constraint(equalTo: topAnchor, constant: 10),
+            scoreTopicLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10),
+            scoreTopicLabel.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -10),
```

Comment:
O único detalhe aqui seria com relação ao tamanho do texto no `scoreTopicLabel`. Se for um texto longo, ou se a tela for muito pequena, pode ser que ele extrapole a largura e acabe passando por cima do `scorePointsView`, já que não foi definida uma constraint para o seu tamanho, ou para a sua `trailingAnchor` .

Definir uma constraint como essa resolveria:

```
scoreTopicLabel.trailingAnchor.constraint(equalTo: scorePointsView.leadingAnchor, constant: -10)
```